administrator/components/com_fields/src/Model/FieldModel.php

```diff
@@ -583,14 +583,9 @@ public function getForm($data = [], $loadData = true)
         // In case we are editing a field, field type cannot be changed, so some extra handling below is needed
         if ($fieldId) {
             $fieldType = $form->getField('type');
-
-            if ($fieldType->value == 'subform') {
-                // Only Use In subform should not be available for subform field type, so we remove it
-                $form->removeField('only_use_in_subform');
-            } else {
-                // Field type could not be changed, so remove showon attribute to avoid js errors
-                $form->setFieldAttribute('only_use_in_subform', 'showon', '');
-            }
+            
+            // Field type could not be changed, so remove showon attribute to avoid js errors
```

Comment:
```suggestion

            // Field type could not be changed, so remove showon attribute to avoid js errors
```
Fix PHPCS failing due to white space in empty line.